vercel/resource_project.go

```diff
@@ -1472,6 +1487,7 @@ func convertResponseToProject(ctx context.Context, response client.ProjectRespon
 		ResourceConfig:                      resourceConfig,
 		NodeVersion:                         types.StringValue(response.NodeVersion),
 		OnDemandConcurrentBuilds:            types.BoolValue(response.ResourceConfig.ElasticConcurrencyEnabled),
+		BuildMachineType:           		 types.StringValue(*response.ResourceConfig.BuildMachineType),
```

Comment:
Don't forget to convert `null` to `standard`? Also, dereferencing the pointer he could panic.